djangocms_attributes_field/fields.py

```diff
@@ -126,19 +126,7 @@ def formfield(self, **kwargs):
         defaults["excluded_keys"] = self.excluded_keys
         return super().formfield(**defaults)
 
-    # This was added to keep backwards compatibility with Django 2.2 while also
-    # avoiding a RemovedInDjango30Warning caused by the deprecation of
-    # context argument
-    if DJANGO_2_2:
-        def from_db_value(self, value,
-                          expression=None, connection=None, context=None):
-            self._from_db_value(value)
-    else:
-        def from_db_value(self, value,
-                          expression=None, connection=None):
-            self._from_db_value(value)
-
-    def _from_db_value(self, value):
+    def from_db_value(self, value, expression=None, connection=None, context=None):
```

Comment:
```suggestion
    def from_db_value(self, value, expression=None, connection=None):
```
Isn't the whole point of this PR to remove the optional context argument? Or am I missing something?

Reply:
You are right! I missed it while trying to fix the test. The latest commit should fix it.